templates/webapi/test/overview.html.ep

```diff
@@ -152,7 +152,8 @@
                     </div>
                     <div class="col-2 align-self-center">
                          <%= help_popover('Help for the <em>Module</em> filter' => '
```

Comment:
```suggestion
                         <%= help_popover('Help for the module filters' => '
```
Or just add a help icon after each heading. That would likely be the best.

Reply:
Good idea ..